common/config/rush/command-line.json

```diff
@@ -0,0 +1,64 @@
+{
+  "customCommands": [
+    {
+      "name": "blah",
+      "description": "Do some really, really cool custom stuff! This is the best command!"
+    }
+  ],
+
+  "customOptions": {
+    "--profile": {
+      "optionType": "enum",
+      "description": "Changes the build profile, affecting the build output",
+      "supportedCommands": [ "build", "rebuild" ],
+      "enumValues": [
+        {
+          "name": "vso",
+          "description": "For running on CI servers"
```

Comment:
>"description": "For running on CI servers" [](start = 10, length = 42)

How does ts-command-line expose this?

Reply:
Maybe Rush could just internally make them bullets and append them to the overall description?

---
In reply to: [146730671](https://github.com/Microsoft/web-build-tools/pull/396#discussion_r146730671) [](ancestors = 146730671)